core/app/models/spree/order_credit_sorter.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Spree
+  class OrderCreditSorter
+    def initialize(credits, _order)
+      @credits = credits
+    end
+
+    def sort
```

Comment:
@kennyadsl - I went with `sort` as the method name which felt like a slightly clearer verb to me (personally). Let me know if you think `call` would be more inline with the overall conventions of Solidus, though?

Reply:
@waiting-for-dev What do you think? I'm still for using `call` for consistency, but we still don't have a real convention in Solidus.

Reply:
Yeah, I'm more in favor of using `#call`. That's the convention we want to use for objects that do a single action, where the class name describes that action.